src/tainting/Dataflow_tainting.ml

```diff
@@ -738,10 +738,16 @@ let resolve_poly_taint_for_java_getters env lval st =
               taints
               |> Taints.map (fun taint ->
                      match taint.orig with
-                     | Arg arg ->
+                     | Arg ({ offset; _ } as arg) when not (List.mem n offset)
+                       ->
+                         (* If the offset we are trying to take is already in the
```

Comment:
Maybe add the example `while (true) { x = x.getX(); }` to the comment, it's obvious now it may no longer be in X months.